main.nf

```diff
@@ -1,4 +1,5 @@
-params.cores = 4
+params.cores = 1 
+params.memory = '4 GB'
```

Comment:
I ended up adding a memory parameter because it seems sample/system dependent. I prefer to keep it low and then configure it higher on samples or systems where necessary but either way works.